Webdev4.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>Leaflet Map</title>
+    <H1>Map made with Leaflet</H1>
+    <H3>GIS 4533 <br>Ryan Peterson</H3>
+    <a href="https://leafletjs.com/">I built this with Leaflet</a>
+    <!-- External Stylesheets -->
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.5.1/dist/leaflet.css" />
+    <script src="https://unpkg.com/leaflet@1.5.1/dist/leaflet.js"></script>
+    <style>
+        body {
+            padding: 0;
+            margin: 0;
+        }
+
+        html,
+        body,
+        #map {
+            height: 100%;
+            width: 100%;
+        }
+    </style>
+
+
+</head>
+
+<body>
+    <!-- Our web map and content will go here -->
+    <!--Deleted the old version and added the latest to the head And adding airport features   -->
+    <script src="https://unpkg.com/leaflet@1.5.1/dist/leaflet.js"></script>
+    <div id="map">
+    </div>
+    <!-- Add JavaScript Library -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script>
+        // Create variable to hold map element, give initial settings to map
+        var map = L.map('map', {
+            center: [46.7867, -92.1005],
+            zoom: 11
+        });
+        // Add OpenStreetMap tile layer to map element
+
+        // Update Basemap to CartoDB Positron
+        var CartoDB_Positron = L.tileLayer('https://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}{r}.png', {
+            attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors &copy; <a href="https://carto.com/attributions">CARTO</a>',
+        }).addTo(map);
+
+        // load GeoJSON from an external file        
+        $.getJSON("duluth_precincts_WGS84.json", function(data) {
+
+            // add GeoJSON layer to the map once the file is loaded
+            L.geoJson(data, {
+                style: function(feature) {
+
+                    // calculate third party percent of vote using function
+                    var PartyPct = calculatePartyPct(feature);
```

Comment:
Your map breaks on this line and the reason is that you don't have a function called `calculatePartyPct`. The one you have below (starting on line 121) is called `calculateThirdPartyPct`